catalyst/dl/scripts/swa.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+from argparse import ArgumentParser
+from pathlib import Path
+
+import torch
+
+from catalyst.dl.utils.swa import generate_averaged_weights
+
+
+def build_args(parser: ArgumentParser):
+    """Builds the command line parameters."""
+    parser.add_argument("--logdir", type=Path, help="Path to models logdir")
```

Comment:
could we make `--logdir` optional? `None` by default
so you could use this script like
```
script --models-mask=/some/path/to/checkpoints*.pth
```
?